data/json/bionics.json

```
@@ -1041,8 +1041,8 @@
     "description": "Your nervous system has been augmented with bionic processors, allowing you to dodge attacks beyond normal human capability, including bullets.",
     "occupied_bodyparts": [ [ "TORSO", 12 ], [ "ARM_L", 3 ], [ "ARM_R", 3 ], [ "LEG_L", 5 ], [ "LEG_R", 5 ], [ "FOOT_L", 1 ], [ "FOOT_R", 1 ] ],
     "flags": [ "BIONIC_TOGGLED" ],
-    "act_cost": 1,
-    "react_cost": 1,
+    "act_cost": "6 J",
+    "react_cost": "6 J",
     "time": 10
```

You said Uncanny Dodge is 6W, so it may should be set to 1 here?
```suggestion
     "time": 1
```